search-commons/src/main/java/no/unit/nva/search2/ResourceAwsClient.java

```diff
@@ -87,6 +87,8 @@ private Stream<QueryBuilderWrapper> createQueryBuilderStream(ResourceAwsQuery qu
                 ? QueryBuilders
                     .multiMatchQuery(query.getValue(SEARCH_ALL).toString(), extractFields(field))
                     .operator(Operator.AND)
+                : query.noLucineParameter()
```

Comment:
hard to read this kind of conditional operator when it is nested

Reply:
Dont really understand it, but if you know it works...

Reply:
I agree...